reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/Analyzer.scala

```diff
@@ -130,7 +130,7 @@ private[analyze] class TraitAnalyzer extends Analyzer {
   override def analyzeNewClassMethods(oldclazz: ClassInfo, newclazz: ClassInfo): List[Problem] = {
     val res = collection.mutable.ListBuffer.empty[Problem]
 
-    for (newmeth <- newclazz.concreteMethods if !oldclazz.hasStaticImpl(newmeth)) {
+    for (newmeth <- newclazz.concreteMethods if newmeth.isDeferred && !oldclazz.hasStaticImpl(newmeth)) {
```

Comment:
`newclazz.concreteMethods` includes all concrete methods of a trait at the Scala level, not the JVM level. They can either be default interface methods (`newmeth.isDeferred == false`) or old-trait-style abstract methods (`newmeth.isDeferred == true`) with a separate implementation method that needs to be called by all implementors of the interface.